vulnerabilities/tests/test_data_migrations.py

```diff
@@ -533,3 +533,38 @@ def test_merge_rows(self):
         ]
 
         assert severities == expected
+
+
+class RemoveCorrupteAdvisories(TestMigrations):
+    app_name = "vulnerabilities"
+    migrate_from = "0036_alter_package_package_url_and_more"
+    migrate_to = "0037_auto_20230120_1204"
```

Comment:
Can we give this a better name?
What about renaming the file to `0037_remove_corrupted_advisories_with_incorrect_refs_and_severity

Reply:
Thanks! done